dashboard/app/views/home/_privacy.md

```diff
@@ -151,7 +151,7 @@ Code.org does not directly collect or store payment instruments. Donations to Co
 <a id="studentemails">
 ## Student Email Addresses
 
-When User accounts use a Student email address for login, Code.org does not store the email address provided by those Users in a retrievable format. Instead, we immediately create and store only a one-way hashed version of the email address (which cannot be converted back into the original address), and use it only for the purposes of login, account management, and password recovery. In fact, when creating a Student account or during login, the actual account email address is never even transmitted to Code.org's servers. The only circumstance when Code.org's servers receive a Student's Code.org email address is if the Student forgets their password and asks to reset it. At that point, the Student is prompted to enter their email, which is used to send them a password reset link.
+When User accounts use a Student email address for login, Code.org does not store the email address provided by those Users in a retrievable format. Instead, we immediately create and store only a one-way hashed version of the email address (which cannot be converted back into the original address), and use it only for the purposes of login, account management, and password recovery. In fact, when creating or signing into a Student account, the actual account email address is never even transmitted to Code.org's servers. The only circumstance when Code.org's servers receive a Student's Code.org email address is if the Student forgets their password and asks to reset it. At that point, the Student is prompted to enter their email, which is used to send them a password reset link.
 
```

Comment:
Line 163: we obtain prior consent from the `Student’s` teacher

Not capitalized in this PR, capitalized in the other one